gateway/gateway.go

```diff
@@ -461,7 +461,8 @@ func (g *gatewayImpl) heartbeat() {
 
 		case <-heartbeatTicker.C:
 			if g.lastHeartbeatSent.After(g.lastHeartbeatReceived) {
-				g.config.Logger.Warn("ACK of last heartbeat not received, connection went zombie")
+				zombieFor := g.lastHeartbeatSent.Sub(g.lastHeartbeatReceived)
+				g.config.Logger.Warn("ACK of last heartbeat not received, connection went zombie", slog.Duration("zombieFor", zombieFor))
```

Comment:
nit: key should be in snake case 🙂 

```suggestion
				g.config.Logger.Warn("ACK of last heartbeat not received, connection went zombie", slog.Duration("zombie_for", zombieFor))
```